language/tools/move-package/tests/test_sources/resolution/package_hooks/Move.toml

```diff
@@ -3,4 +3,4 @@ name = "test"
 version = "0.0.0"
 
 [dependencies]
-Pkg = { custom = "localhost:8080", address = "0x1", package = "framework" }
```

Comment:
Was the `package = "framework"` field used before?  From what I could tell, the answer was no, and the field was triggering a warning during `Move.toml` parsing so I removed it, but I can add it back.

Reply:
I think the idea was for renaming of dependencies if needed, so something like `PkgA = { url = ..., package = "pkg" } PkgB = { ..., package = "pkg" }`

Reply:
Hmm, do you know if that is supposed to still work or is it a holdover? The manifest parser doesn't recognise the field, and it doesn't get passed to the package hook (I also looked at the [`AptosPackageHook`](https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos/src/move_tool/package_hooks.rs))

Reply:
no idea, sorry :)